core/domain/suggestion_services.py

```diff
@@ -1265,3 +1265,18 @@ def _update_suggestion_counts_in_community_contribution_stats(
     """
     _update_suggestion_counts_in_community_contribution_stats_transactional(
         suggestions, amount)
+
+
+def update_translation_suggestion(suggestion_id, translation_html):
+    """Updates the change.translation_html of translation suggestion with the
+        given suggestion_id.
+
+    Args:
+        suggestion_id: str. The id of the suggestion to be updated.
+        translation_html: str. The new translation html.
+    """
+    suggestion = get_suggestion_by_id(suggestion_id)
+
+    suggestion.change.translation_html = translation_html
```

Comment:
Can you please use `html_cleaner.clean(translation_html)` here
```suggestion
    suggestion.change.translation_html = html_cleaner.clean(translation_html)
```

Reply:
I updated the parms to generalise for all suggestions. In case for this, don't we need to implement a generic method as it's been raised in this [issue](https://github.com/oppia/oppia/issues/12482)